libs/perun/services/src/lib/table-checkbox.service.ts

```diff
@@ -112,7 +107,7 @@ export class TableCheckbox {
           });
         } else {
           dataSource.sortData(dataSource.data, sort).forEach(row => {
-            if (dataSource.sortData(dataSource.filteredData, sort).includes(row)) {
+            if (dataSource.filteredData.includes(row)) {
```

Comment:
The condition `dataSource.filteredData.includes(row)` doesn't work correctly. I already tried it in issue #597 and concretely for this line you can try the use case described in the issue (use case for attributes with filter in Perun admin).